echo/.devcontainer/docker-compose.yml

```
@@ -30,13 +32,28 @@ services:
+      - 7474:7474  # Neo4j Browser
+      - 7687:7687  # Neo4j Bolt protocol
     environment:
       POSTGRES_DB: dembrane
       POSTGRES_USER: dembrane
       POSTGRES_PASSWORD: dembrane
     volumes:
       - ./postgres_data:/var/lib/postgresql/data
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+
+  neo4j:
+    image: neo4j:latest    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)